src/gamestate.ts

```diff
@@ -1,12 +1,12 @@
-interface GameState {
+export interface GameState {
   board: Array<Array<Cell>>;
 }
 
-interface Cell {
+export interface Cell {
   value: CellValue;
 }
 
-enum CellValue {
+export enum CellValue {
```

Comment:
use union instead of enum

Reply:
will be resolved with https://github.com/marmelab/Hex/pull/3#discussion_r801038301